### PR TITLE
Added navigation-bundle

### PR DESCRIPTION
**Pull request by twanhaverkamp**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by Nyholm** [comment]
Hey. You do not need to create multiple PRs. Just push a new commit to your exiting branch.

**Comment by twanhaverkamp** [comment]
Hi @Nyholm,

First of all thanks for your comment! I know that the pullrequest will be updated automatically, but I had to make a few changes in my package.

I wasn't able to rerun the Travis build without creating a new pullrequest.

File: twanhaverkamp/navigation-bundle/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "TwanHaverkamp\\CoreBundle\\CoreBundle": ["all"],
```
Comment (Nyholm) [inline twanhaverkamp/navigation-bundle/1.0/manifest.json]: We should not add other bundles here.

File: twanhaverkamp/navigation-bundle/1.0/config/packages/navigation.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+services:
+    # if you would like to load navigation items from a YAML file see:
+    # https://github.com/twanhaverkamp/navigation-bundle#step-1-pass-the-yamlprovider-as-argument
+    TwanHaverkamp\NavigationBundle\Twig\NavigationExtension:
```
Comment (Nyholm) [inline twanhaverkamp/navigation-bundle/1.0/config/packages/navigation.yaml]: Why dont you register this service in your bundle?

**Comment by twanhaverkamp** [comment]
Hi @Nyholm,

Thanks for your feedback! I made a few changes within the bundle and updated the recipe.

**Comment by Nyholm** [comment]
Hm. The bot disapproves this PR. 
That means that we should not merge this PR since it does not have any config. The `post-install` messages (which I actually liked) should rather be in your documentation.

**Comment by twanhaverkamp** [comment]
Hi Tobias,

Thanks for your feedback! Unfortunately, I liked the post install message. It’s a nice way to get a user up and running.

Are you attending SymfonyCon Amsterdam this year?

**Pull request closed without merging**